.github/workflows/installations.yml

```diff
@@ -0,0 +1,102 @@
+name: installations
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    paths:
+    - 'FirebaseInstallations**'
+    - '.github/workflows/installations.yml'
+    - 'Gemfile'
+  schedule:
+    # Run every day at 11pm (PST) - cron uses UTC times
+    - cron:  '0 7 * * *'
+
+jobs:
+  pod-lib-lint:
+    runs-on: macOS-latest
+    env:
+      plist_secret: ${{ secrets.GHASecretsGPGPassphrase1 }}
+    strategy:
+      matrix:
+        target: [ios, tvos, macos]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup Bundler
+      run: scripts/setup_bundler.sh
+    - name: Install GoogleService-Info.plist
+      run: |
+        mkdir -p FirebaseInstallations/Source/Tests/Resources
+        scripts/decrypt_gha_secret.sh scripts/gha-encrypted/Installations/GoogleService-Info.plist.gpg \
+          FirebaseInstallations/Source/Tests/Resources/GoogleService-Info.plist "$plist_secret"
+    - name: Get boolean for secrets available
+      id: secrets
+      run: echo "::set-output name=val::$([[ -z $plist_secret ]] && echo "0" || echo "1")"
+    - name: Build and test
+      run: |
+       export FIS_INTEGRATION_TESTS_REQUIRED=${{ steps.secrets.outputs.val }}
+       export FIS_ALLOWS_INCOMPATIBLE_IID_VERSION="1"
+       scripts/third_party/travis/retry.sh scripts/pod_lib_lint.rb FirebaseInstallations.podspec \
+         --platforms=${{ matrix.target }} --ignore-local-podspecs=FirebaseInstanceID.podspec
+
+  # TODO - Catalyst is disabled because `pod gen` does not have a way to get some dependent pods
```

Comment:
I think we can introduce an additional flag to disable specific tests requiring a specific IID version. We can address this later.

Reply:
SG - or do we need to keep testing the old version of IID?